Auth0/Authentication.swift

```diff
@@ -774,15 +774,15 @@ public extension Authentication {
      ```
 
      - parameter credentials: the client's credentials obtained on auth
-     - parameter scope: scopes to request for the new tokens. By default is nil which will ask for the same ones requested during Auth.
+     - parameter scope: scopes to request for the new tokens. By default is nil which will ask for the same ones requested during Auth
+     - parameter callback: callback with the user's credentials or the cause of the error.
      - important: This method only works for a refresh token obtained after auth with OAuth 2.0 API Authorization.
-     - returns: a request that will either yield an error or Auth0 user's credentials
      */
     func renewExpired(_ credentials: Credentials, scope: String? = nil, callback: @escaping (Error?, Credentials?) -> Void) {
-        guard
-            let refreshToken = credentials.refreshToken,
-            let expiresIn = credentials.expiresIn, expiresIn < Date(timeIntervalSinceNow: -60)
-            else { return callback(AuthenticationError(string: "missing refresh_token or have not expired", statusCode: 400), nil) }
+        guard let refreshToken = credentials.refreshToken else {
+            return callback(AuthenticationError(string: "missing refresh_token", statusCode: 400), nil)
+        }
+        guard let expiresIn = credentials.expiresIn, expiresIn < Date(timeIntervalSinceNow: -60) else { return callback(nil, credentials) }
```

Comment:
What does the `-60` means? Now that I see this, it reminds me that some tokens might be deemed invalid wrongly depending on the device's clock. Remember the leeway thing we support in our jwt libs? Maybe we should open a method to customize it too for this case. It should not be an issue for most of the users but some of them might have it.